src/model/Repositories/PasswordResetTokensRepository.php

```diff
@@ -11,11 +11,12 @@ class PasswordResetTokensRepository extends Repository
 {
     protected $tableName = 'password_reset_tokens';
 
-    final public function add(IRow $user, $expire = '+5 hours')
+    final public function add(IRow $user, $source = null, $expire = '+5 hours')
```

Comment:
This one will introduce breaking change, but the `source` parameter should probably be in front of `expire` so we can live with that :).